ra/ra_test.go

```diff
@@ -155,14 +156,51 @@ func numAuthorizations(o *corepb.Order) int {
 }
 
 type DummyValidationAuthority struct {
-	performValidationRequest             chan *vapb.PerformValidationRequest
-	PerformValidationRequestResultError  error
-	PerformValidationRequestResultReturn *vapb.ValidationResult
+	doDCVRequest chan *vapb.PerformValidationRequest
+	doDCVError   error
+	doDCVResult  *vapb.ValidationResult
+
+	doCAARequest  chan *vapb.IsCAAValidRequest
+	doCAAError    error
+	doCAAResponse *vapb.IsCAAValidResponse
 }
 
 func (dva *DummyValidationAuthority) PerformValidation(ctx context.Context, req *vapb.PerformValidationRequest, _ ...grpc.CallOption) (*vapb.ValidationResult, error) {
-	dva.performValidationRequest <- req
-	return dva.PerformValidationRequestResultReturn, dva.PerformValidationRequestResultError
+	dva.doDCVRequest <- req
```

Comment:
dva.DoDCV has this same line, so I don't think it's necessary to replicate it here
```suggestion
```